frontend/pages/admin/IntegrationsPage/cards/MdmSettings/MicrosoftGraphPage/_styles.scss

```
@@ -0,0 +1,24 @@
+.microsoft-graph-page {
+  @include vertical-page-layout;
+
+  &__content-container {
+    @include flex-column-32px-gap;
+    max-width: $settings-form-max-width;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)